PhysicsTools/PatAlgos/python/slimming/puppiForMET_cff.py

```diff
@@ -59,8 +59,6 @@ def makePuppiesFromMiniAOD( process, createScheduledSequence=False ):
     process.puppiForMET.candName = cms.InputTag('packedPFCandidates')
     process.puppiForMET.photonName = cms.InputTag('slimmedPhotons')
     process.puppiForMET.runOnMiniAOD = cms.bool(True)
-    setupPuppiPhotonMiniAOD(process)
```

Comment:
Is it then still needed to ```import setupPuppiPhoton,setupPuppiPhotonMiniAOD```?

Reply:
if not needed this should really be dropped, as it can introduce bad circular dependencies against which we already fought in the past

Reply:
In the master it is dropped from all configs I believe

Reply:
what about this?

Reply:
I dropped this line (as was done in the master in #27929), because what is computed is not used by puppi and thus simplifies running with customized PUPPI tune the MiniAOD->NanoAOD step.